cmd/kube-proxy/app/server_others_test.go

```diff
@@ -150,7 +159,11 @@ func Test_getProxyMode(t *testing.T) {
 			modules:       c.kmods,
 			kernelVersion: c.kernelVersion,
 		}
-		canUseIPVS, _ := ipvs.CanUseIPVSProxier(khandler, ipsetver)
+		scheduler := cases[i].scheduler
+		if scheduler == "nil" {
```

Comment:
I think this should just be an empty string check instead?

Reply:
Oops ya that's correct.